cfgov/jinja2/v1/job-description-page/index.html

```diff
@@ -81,16 +82,13 @@ <h2 id="interested">Interested in Applying?</h2>
                          Social Media icons are slightly high
                          relative to Interested in Applying heading. #}
                 {{ social_media.render( {
-                    'title':            page.job_title,
-                    'twitter_hashtags': page.job_twitter_hash_tag or 'usajobs'
+                    'title':            page.title,
+                    'twitter_hashtags': page.twitter_hash_tag or 'usajobs'
```

Comment:
Hmmmm. I don't think so -- as long as it generates a customized tweet with the appropriate link and job title, then it should be able to be automated.